### PR TITLE
Wsegaicd support

### PR DESCRIPTION
**Pull request by tklov11**
Support for WSEGSICD and WSEGAICD was added in response to issue #199. Functionality is included in
- ecl2df/compdat.py
- ecl2df/common.py
-ecl2df/opmkeywords/WSEGSICD
-ecl2df/opmkeywords/WSEGAICD

Two tests are added to
- tests/test_compdat.py

New include files for testing are added to
- tests/data/reek/eclipse/include/schedule/

### CODE REVIEW

File: ecl2df/common.py [hunk 1]
```diff
@@ -54,6 +54,8 @@
     "WELOPEN",
     "WELSEGS",
     "WELSPECS",
+    "WSEGSICD",
```
Comment (berland) [inline ecl2df/common.py:57]: Make list alphabetical.

File: ecl2df/compdat.py [hunk 2]
```diff
@@ -173,14 +186,34 @@ def deck2dfs(deck, start_date=None, unroll=True):
...
+
+    if unroll and not wsegaicd_df.empty:
+        wsegaicd_df = unrolldf(wsegaicd_df, "SEGMENT1", "SEGMENT2")
+
     if "KEYWORD_IDX" in compdat_df.columns:
         compdat_df.drop(["KEYWORD_IDX"], axis=1, inplace=True)
 
-    return dict(COMPDAT=compdat_df, COMPSEGS=compsegs_df, WELSEGS=welsegs_df)
+    if "KEYWORD_IDX" in wsegsicd_df.columns:
+        wsegsicd_df.drop(["KEYWORD_IDX"], axis=1, inplace=True)
```
Comment (berland) [inline ecl2df/compdat.py:205]: I see this drop is not done for WELSEGS and COMPSEGS, for which WSEG*ICD are analogues. So potentially it should not be done for WSEG*ICD or it should also be done for WELSEGS and COMPSEGS?

File: tests/test_compdat.py [hunk 3]
```diff
@@ -428,3 +438,110 @@ def test_defaulted_compdat_i_j(tmpdir):
...
+    compdat_df = compdfs["COMPDAT"]
+    welsegs = compdfs["WELSEGS"]
+    compsegs = compdfs["COMPSEGS"]
+    wsegaicd = compdfs["WSEGAICD"]
+    assert "WELL" in compdat_df
+    assert len(compdat_df) == 1
+    assert compdat_df["WELL"].unique()[0] == "OP_6"
+
+    # Check that we have not used the very long opm.io term here:
+    assert "CONNECTION_TRANSMISSIBILITY_FACTOR" not in compdat_df
```
Comment (berland) [inline tests/test_compdat.py:511]: This line should not be needed here, as it is tested further up? 
Can you shorten the tests here to only those that are not tested elsewhere?

(test code also required future maintenance, and superfluous test code increases maintenance burden)

File: ecl2df/compdat.py (hunk 2 above)
Comment (tklov11) [reply ecl2df/compdat.py:205]: Hmm, her har det skjedd noe feil. La meg sjekke litt og gjøre en pull-request til.

Comment (tklov11) [reply ecl2df/compdat.py:205]: Well, WSEGxICD is analogous to COMPDAT with only one record, so I used COMPDAT as a template. WELSEGS and COMPSEGS have two records. KEYWORD_IDX is not defined in WELSEGS and COMPSEGS and therefore not dumped. If KEYWORD_IDX should have been defined/dumped for WELSEGS and COMPSEGS I think you should discuss with the developer for those keywords?

File: ecl2df/common.py (hunk 1 above)
Comment (tklov11) [reply ecl2df/common.py:57]: Done

File: tests/test_compdat.py (hunk 3 above)
Comment (tklov11) [reply tests/test_compdat.py:511]: Ja, fjerner alt som er gjort før men legger inn en test for WSEGSICD i tillegg til WSEGAICD som ligger der fra tidligere.

File: ecl2df/compdat.py (hunk 2 above)
Comment (berland) [reply ecl2df/compdat.py:205]: Thanks, that sounds reasonable. Maybe that information should be included as a code comment.

**Review by berland — APPROVED** [review]
Good! Thanks for the contribution

**Pull request merged**